### PR TITLE
init project

### PR DESCRIPTION
**Pull request by meimei0111**

### CODE REVIEW

**Comment by meimei0111** [comment]
ready

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
```
Comment (saobv-2223) [inline app/javascript/packs/application.js]: remove all the comment line
image

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb]: image

File: config/credentials.yml.enc [hunk 3]
```diff
@@ -0,0 +1 @@
+SIMv/aXefrwznH+N+dSHBEkISuaVrKL7wPJY9eF57Z9pOxPc9fjWOnavl3f8maM4bd82AIN3cZcckfB544UioSfXIKhPIw8YwaJOmlk+6WyoRs7BmRhyMV/S4vztivYbnrdTudWJMHdCnyG3qdibp/l9pTDdwT2ogLcHu+xUkD3ztrdJyMQAS1xQcmXsjv/ck6g9Zem9OYUYc7o+V6+i2uN/bh8YJKBmvIi035zNURipfXnEe/JEnusmlxsO7xZB6zKTCE8OLjTWvaIGas8oPlWv74hYBuNTNukiKkgiUrTwtVPE6mUPcEG4PscPBi9nIlGtT84XJIh92BtLKoZRJcrhJt08Rg9Qe9fzj1mtdTZMoqYntAxx+2Z6Ir7uxIOr/OZXie9ZCE6hMUZoYj0/b0qspxp71EV1yztG--Yi80BEw30sBfsJTl--GPAUR/tk27JFjjgAlqJ5FA==
```
Comment (saobv-2223) [inline config/credentials.yml.enc]: image

File: app/javascript/packs/application.js (hunk 1 above)
Comment (meimei0111) [reply app/javascript/packs/application.js]: done

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (meimei0111) [reply app/views/layouts/application.html.erb]: done

File: config/credentials.yml.enc (hunk 3 above)
Comment (meimei0111) [reply config/credentials.yml.enc]: done

**Comment by meimei0111** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Comment by ngocvt-0484** [comment]
commented

**Pull request merged**